.github/workflows/ci-testing-deploy.yml

```diff
@@ -1087,7 +1087,47 @@ jobs:
           path: tests/e2e/screenshots
       - name: cleanup
         if: always()
-        run: ./ci/github/system-testing/swarm-deploy.bash clean_up
+        run: ./ci/github/system-testing/e2e.bash clean_up
+
+  system-test-environment-setup:
+    name: System-testing environment setup
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        python: [3.6]
+        os: [ubuntu-16.04, ubuntu-18.04, ubuntu-20.04]
```

Comment:
@sanderegg or @Surfict  ? Can you answer that?
I guess for the moment we do not have all clusters's OS still in sync.. @Surfict  ? 

Besides, yes, it would be nice to get all up-to-date

Reply:
Updated.